excelerator.md

```diff
@@ -2,3 +2,25 @@
 permalink: /excelerator.html
 ---
 
+# Excelerator  
+
+[http://excelerator.loci.cat](http://excelerator.loci.cat)
+
+## What 
+
+A data reapportionment tool. This tool will enable users to reapportion data referenced using one Loc-I data set to another Loc-I dataset, e.g. population count by ASGS Meshblocks reapportioned to Geofabric contracted catchments. 
```

Comment:
reapportion 'values' or 'data values'

Reply:
reapportioning values would be interesting... noted `data values`